test/rulesets/Base/array.jl

```diff
@@ -4,6 +4,14 @@
     test_rrule(reshape, rand(4, 5), 2, :)
 end
 
+@testset "repeat" begin
+    test_rrule(repeat, rand(4, ))
+    test_rrule(repeat, rand(4, ), 2)
+    test_rrule(repeat, rand(4, 5))
+    test_rrule(repeat, rand(4, 5); fkwargs = (inner = (2,1), outer = (2,2)))
+    test_rrule(repeat, rand(4, 5), 2, 3)
```

Comment:
Some test cases which you may wish to add, with `@test_broken` if they are broken:
```
repeat([1 2; 3 4], outer=3)  # perhaps works accidentally, compare  repeat([1 2; 3 4], inner=(1,1), outer=2) 
repeat([1 2; 3 4], inner=(2,4), outer=(1,1,1,3))
repeat([1,2,3], 2, 3)

# zero-arrays
repeat(fill(1), 2)
repeat(fill(1), 2, 3)
repeat(fill(1), inner=2)
repeat(fill(1), inner=2, outer=3)